crates/fuel-gas-price-algorithm/src/v1.rs

```diff
@@ -151,13 +146,128 @@ pub struct AlgorithmUpdaterV1 {
     pub second_to_last_profit: i128,
     /// The latest known cost per byte for recording blocks on the DA chain
     pub latest_da_cost_per_byte: u128,
-
+    /// Activity of L2
+    pub l2_activity: L2ActivityTracker,
     /// The unrecorded blocks that are used to calculate the projected cost of recording blocks
     pub unrecorded_blocks: BTreeMap<Height, Bytes>,
 }
 
-/// A value that represents a value between 0 and 100. Higher values are clamped to 100
+/// Because the DA gas price can increase even when no-one is using the network, there is a potential
+/// for a negative feedback loop to occur where the gas price increases, further decreasing activity
+/// and increasing the gas price. The `L2ActivityTracker` is used to moderate changes to the DA
+/// gas price based on the activity of the L2 chain.
+///
+/// For each L2 block, the activity is calculated as a percentage of the block capacity used. If the
+/// activity is below a certain threshold, the activity is decreased. The activity exists on a scale
+/// between 0 and the sum of the increase, hold, and decrease buffers.
+///
+/// e.g. if the decrease range is 50, the hold range is 50, and the increase range is 50:
+///
+/// 0<-- decrease range -->50<-- hold range -->100<-- increase range -->150
```

Comment:
Maybe a little variancy in sizes could make the example more clear.

For example, something like below?

```suggestion
/// e.g. if the decrease range is 20, the hold range is 40, and the increase range is 60:
///
/// 0<-- decrease range -->20<-- hold range -->60<-- increase range -->120
```